clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -38,8 +37,7 @@ trait IoUtil {
     Files
       .walk(directory, FileVisitOption.FOLLOW_LINKS)
       .sorted(Comparator.reverseOrder())
```

Comment:
For very deep trees this will be expensive. Using a file visitor instead would avoid having to read all file tree paths at once.

Also I wonder why we're using `FOLLOW_LINKS` here.